terraform/install.sh

```diff
@@ -86,7 +86,7 @@ promptForProject() {
                  --flatten="bindings[].members" \
                  --format="table(bindings.members)" \
                  --filter="bindings.role:roles/owner" 2> /dev/null | grep $acct | cat)
-      if [[ -n "$iam_test" ]]; then
+    if [[ -n "$iam_test" ]]; then
```

Comment:
was this intentional?

Reply:
Oh I just want to align the indentations.

Reply:
sounds good